### PR TITLE
mention embedded pdb

### PR DESCRIPTION
**Pull request by bruno-garcia**
Relates to: https://github.com/MicrosoftDocs/visualstudio-docs/issues/2204

### CODE REVIEW

File: docs/create-packages/Symbol-Packages-snupkg.md [hunk 1]
```diff
@@ -18,6 +18,9 @@ ms.reviewer:
 
 A good debugging experience relies on the presence of debug symbols as they provide critical information like the association between the compiled and the source code, names of local variables, stack traces, and more. You can use symbol packages (.snupkg) to distribute these symbols and improve the debugging experience of your NuGet packages.
 
+> Note that symbol package isn't the only strategy to make the debug symbols available to the consumers of your library. It's also [possible to `embed`](https://docs.microsoft.com/en-us/dotnet/core/deploying/single-file#include-pdb-files-inside-the-bundle) them in the `dll` or `exe` with the following project property:
```
Comment (bruno-garcia) [inline docs/create-packages/Symbol-Packages-snupkg.md:21]:
```suggestion
> Note that symbol package isn't the only strategy to make the debug symbols available to the consumers of your library. It's also [possible to `embed`](https://docs.microsoft.com/dotnet/core/deploying/single-file#include-pdb-files-inside-the-bundle) them in the `dll` or `exe` with the following project property:
```

**Comment by bruno-garcia** [comment]
> Line 4, Column 12: [Suggestion-ms-author-invalid] Invalid value for 'ms.author', 'cristinamanu' is not a valid Microsoft alias.

Sounds like the author left Microsoft?
Each document must have a handle of an MS employee?
Does this affect my contribution?

**Review by JonDouglas — APPROVED** [review]
Thank you for the contribution Bruno! I apologize well in advance that it took this long to get merged.

**Comment by JonDouglas** [comment]
@bruno-garcia No this does not affect your contribution. It is an artifact of the documentation tooling & requires a mass update from our side. We will clean this up to make contributions more clearer and easier in the future. Sorry about that.

**Comment by bruno-garcia** [comment]
Oh no problem at all, I was just wondering if it could get merged as-is or if I would need to change anything.

Thanks for merging Jon.

**Pull request merged**